plugins/modules/keycloak_user.py

```diff
@@ -0,0 +1,496 @@
+                    - Name of the group
+                type: str
+            state:
+                description:
+                    - Control if the user must be member of this group or not
+                choices: [ "present", "absent" ]
+                default: present
+                type: str
+    credentials:
+        description:
+            - User credentials.
+        default: []
+        type: list
+        elements: dict
+        suboptions:
+            type:
+                description:
+                    - Credential type
+                type: str
+                required: true
+            value:
+                description:
+                    - Value of the credential
+                type: str
+                required: true
+            temporary:
+                description:
+                    - If true, the users require to reset this credentials at next logon.
+                type: bool
+                default: false
+    required_actions:
+        description:
+            - requiredActions user Auth.
+        default: []
+        type: list
+        elements: str
+        aliases:
+            - requiredActions
+    federated_identities:
+        description:
+            - list of IDP of user.
+        default: []
+        type: list
+        elements: str
+        aliases:
+            - federatedIdentities
+    attributes:
+        description:
+            - list user attributes.
+        required: false
+        type: list
+        elements: dict
+        suboptions:
+            name:
+                description:
+                    - Name of the attributes
+                type: str
+            values:
+                description:
+                    - Values for the attribute as list
+                type: list
+                elements: str
+            state:
+                description:
+                    - Control if the attribute must exists or not
+                choices: [ "present", "absent" ]
+                default: present
+                type: str
+    access:
+        description:
+            - list user access.
+        required: false
+        type: dict
+    disableable_credential_types:
+        description:
+            - list user Credential Type.
+        default: []
+        type: list
+        elements: str
+        aliases:
+            - disableableCredentialTypes
+    origin:
+        description:
+            - user origin.
+        required: false
+        type: str
+    self:
+        description:
+            - user self administration.
+        required: false
+        type: str
+    state:
+        description:
+            - Control if the user must exists or not
+        choices: [ "present", "absent" ]
+        default: present
+        type: str
+    force:
+        description:
+            - If true, allows to remove user and recreate it.
+        type: bool
+        default: false
+extends_documentation_fragment:
+    - keycloak
```

Comment:
Always use FQCNs for stuff in community.general. Also add attributes
```suggestion
    - community.general.keycloak
    - community.general.attributes
attributes:
    check_mode:
        support: full
    diff_mode:
        support: full
```